packages/runtime-core/__tests__/apiWatch.spec.ts

```diff
@@ -548,6 +548,98 @@ describe('api: watch', () => {
     expect(cb).not.toHaveBeenCalled()
   })
 
+  // #7030
+  it('should not fire on child component unmount w/ flush: pre', async () => {
+    const visible = ref(true)
+    const cb = jest.fn()
```

Comment:
`jest.fn()` has been replaced with `vi.fn()` on the latest `main` branch.

Reply:
Thanks for reminding.